appveyor.yml

```diff
@@ -1,26 +1,29 @@
+# Warning: This file is automatically synced from https://github.com/ipfs/ci-sync so if you want to change it, please change it there and ask someone to sync all repositories.
+version: "{build}"
+
 environment:
   matrix:
     - nodejs_version: "6"
     - nodejs_version: "8"
 
-init:
-  - git config --global core.autocrlf input
+matrix:
+  fast_finish: true
 
-# cache:
-#  - node_modules
+install:
+  # Install Node.js
+  - ps: Install-Product node $env:nodejs_version
 
-platform:
-  - x64
```

Comment:
This will prevent test on x32. https://github.com/ipfs/js-ipfs/issues/1127 can be closed

Reply:
We don't want to test on x32? I'm unsure if the 32 build is missing in dist.ipfs.io on purpose or if it was just skipped temporary. @whyrusleeping ?

Reply:
Sorry, read the patch wrongly.  This is removing x64, which means appveyor could use a x32 processor, which go-ipfs is not shipping.